src/playercards/customizable/SummonedServitorUpgradeSheet.ttslua

```diff
@@ -31,27 +31,54 @@ existingBoxes = { 1, 1, 1, 1, 1, 2, 3, 5 }
 
 inputBoxes = {}
 
+-- Locations of the skill selectors
+local SLOT_ICON_POSITIONS = {
+    arcane = { x = 0.160, z = 0.65},
+    ally = { x = -0.073, z = 0.65}
+  }
+  
+  local selectedSlots = {
```

Comment:
Since only one of these can be selected at a time (unlike Living Ink) I think a single string value would be cleaner